app/services/mapping/goodsshipment/consignment/ConsignmentBuilder.scala

```diff
@@ -47,9 +47,9 @@ class ConsignmentBuilder @Inject()(
     )
 
     exportsCacheModel.choice match {
-      case AllowedChoiceValues.SupplementaryDec => transportEquipmentBuilder.buildThenAdd(Seq.empty, consignment)
       case AllowedChoiceValues.StandardDec =>
         transportEquipmentBuilder.buildThenAdd(exportsCacheModel.seals, consignment)
+      case _ =>
```

Comment:
I think this change is necessary to match the previous behaniour - see https://github.com/hmrc/customs-declare-exports-frontend/commit/3ef2b037c4f015072957926ad99174d45d253174#diff-10bed3bc431a694eeac25d10b0a49cc8L43

Otherwise we get this for a SupplementaryDec 
<ns3:TransportEquipment>
    <ns3:SequenceNumeric>0</ns3:SequenceNumeric>
</ns3:TransportEquipment>